source/blog/2018-04-20-the-emberjs-times-issue-43.md

```diff
@@ -50,8 +50,16 @@ and give your thoughts and suggestions in the discussion.
 
 <a class="ember-button ember-button--centered href="/blog/2018/04/13/ember-3-1-released.html">Read more</a>
 
+[Ember CLI Update Updated](https://github.com/ember-cli/ember-cli-update#options): Ember CLI Update, which last week was brought in under the official ember-cli GitHub organization, has gotten an update. It now has two new commands that will help you keep your apps updated and using the latest and greatest in Emberland.
```

Comment:
That's a great section - what do you think about making this its own section for this week:

```
## [Ember CLI Updated Updated](https://github.com/ember-cli/ember-cli-update#options)

Ember CLI Update, which last week....
```

Reply:
@Alonski Yes, sounds perfect!